test/unit_tests/js/modules/util/breakpoint-state-spec.js

```diff
@@ -36,32 +34,41 @@ describe( 'breakpoint-state', () => {
   describe( '.getBreakpointState()', () => {
     it( 'should correctly return bpXS state', () => {
       windowResizeTo( 300 );
-      expect( getBreakpointState() ).toStrictEqual( { bpLG: false, bpMED: false, bpSM: false, bpXL: false, bpXS: true } );
+      expect( getBreakpointState() ).toStrictEqual(
+        { bpLG: false, bpMED: false, bpSM: false, bpXL: false, bpXS: true }
```

Comment:
Could stack these, but it's up to your preference.
```
{ 
  bpLG: false, 
  bpMED: false, 
  bpSM: false, 
  bpXL: false, 
  bpXS: true 
}
```

Reply:
Same for additional occurrences in this file.

Reply:
I agree and put each breakpoint state on separate lines